src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/Instrumentation.kt

```diff
@@ -70,7 +70,14 @@ class GoboInstrumentation(
 
             context.query = queryText
             if (queryText.isNotIntrospectionQuery()) {
-                logger.debug(queryText)
+                queryCache?.let { cache ->
+                    val request = context.request
+                    cache.add(request.klientid(), request.korrelasjonsid(), queryText)
+                }
+
+                when (logQueries) {
```

Comment:
Im a sucker for when, men kan en oneline if være mer aktuelt her?

Reply:
fixed